activities-dependency-injection/package.json

```diff
@@ -7,7 +7,8 @@
     "build.watch": "tsc --build --watch",
     "start": "ts-node src/worker.ts",
     "start.watch": "nodemon src/worker.ts",
-    "workflow": "ts-node src/execute-workflow.ts"
+    "workflow": "ts-node src/execute-workflow.ts",
+    "lint": "eslint ."
```

Comment:
I wish we had a template for this too

Reply:
For package.json? I think too many differences to be worthwhile

Reply:
I didn't mean the whole file but it would have been nice to have some shared structure that we can sync.
Definitely not for this PR.